src/assert.cpp

```diff
@@ -175,7 +175,7 @@ TORRENT_EXPORT void print_backtrace(char* out, int len, int max_depth
 	std::array<void*, 50> stack;
 
 	STACKFRAME64 stack_frame;
-	memset(&stack_frame, 0, sizeof(stack_frame));
+	std::memset(&stack_frame, 0, sizeof(stack_frame));
```

Comment:
while you're here, would you mind removing the ``memset()`` and say ``STACKFRAME64 stack_frame = {};`` instead?